lte/gateway/python/magma/mobilityd/ip_allocator_dhcp.py

```diff
@@ -37,14 +42,18 @@
 
 DEFAULT_DHCP_REQUEST_RETRY_FREQUENCY = 10
 DEFAULT_DHCP_REQUEST_RETRY_DELAY = 1
+LEASE_RENEW_WAIT_MIN = 200
 
+DHCP_HELPER_CLI_PATH = f"{environ.get('MAGMA_ROOT')}/lte/gateway/python/scripts/dhcp_helper_cli.py"
```

Comment:
Oh, I missed this. This is fine for dev but I don't think that's going to work with deployed installations of magma. They currently install the script at `/usr/local/bin/dhcp_helper_cli.py` and don't rely on MAGMA_ROOT.

I am not sure what's the best way to fix it, but we should focus on deployments first and only allow dev as a backup. Maybe check if `dhcp_helper_cli.py` is in the user's `PATH` and only try the `MAGMA_ROOT` path as a backup.

Reply:
Maybe we don't even need the fallback. I think it's in the PATH in magma-dev too.

Reply:
It is not in the `PATH` for me, actually. But that should be easy to fix in the meantime, but we can just need a link in the meantime

Reply:
I tested that it works for and will update the file to call the script directly. (I think you need to be in the virtualenv to see it in the PATH).